DataConnectors/Corelight/Connector_LogAnalytics_agent_Corelight.json

```diff
@@ -0,0 +1,148 @@
+{
+    "id": "Corelight",
+    "title": "Corelight",
+    "publisher": "Corelight",
+    "descriptionMarkdown": "The [Corelight](https://corelight.com/) data connector provides the capability to ingest [Corelight Zeek/Bro events](https://www3.corelight.com/zeek-logs-v3.0) into Azure Sentinel. Refer to [Corelight Logs documentation](https://corelight.com/about-zeek/how-zeek-works) for more information.",
+    "additionalRequirementBanner": "This data connector depends on a parser based on Kusto Function to work as expected. Follow the steps to use this Kusto Function alias **Corelight** in queries and workbooks. [Follow steps to get this Kusto Function>](https://aka.ms/sentinel-Corelight-parser)",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "Corelight Events",

```

Comment:
Legend name should be datatype name. Could you please update it  to "Corelight_CL".

Thanks,
Aditi Maheshwari

Reply:
done